norbit-mobileapp/flutter_application_1/test/service_test/sensor_test/lux_service_test.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter_application_1/blocs/sensors/lux_bloc.dart';
+import 'package:flutter_application_1/services/sensors/lux_service.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('LuxService Tests', () {
+    test('Add lux data to bloc', () {
+      final luxBloc = LuxBloc();
+      final luxService = LuxService(luxBloc: luxBloc);
+
+      const testLuxValue = 500; // Replace with a sample lux value for testing
```

Comment:
The comment seems mildly misleading. The comment seems to suggest that the value should be changed for testing, when it's actually emitted as a constant for testing. What about something like this instead?
```suggestion
      // Emit a constant sample lux value for testing
      const testLuxValue = 500;
```